modules/fluid_properties/unit/src/SimpleFluidPropertiesTest.C

```diff
@@ -56,9 +59,23 @@ TEST_F(SimpleFluidPropertiesTest, properties)
   ABS_TEST(_fp->mu_from_p_T(p, T), visc, tol);
   ABS_TEST(_fp->h_from_p_T(p, T), cv * T + p / _fp->rho_from_p_T(p, T), tol);
   ABS_TEST(_fp2->h_from_p_T(p, T), cv * T + p * pp_coef / _fp2->rho_from_p_T(p, T), tol);
+  ABS_TEST(_fp->cp_from_v_e(v, e), cp, tol);
+  ABS_TEST(_fp->cv_from_v_e(v, e), cv, tol);
+  ABS_TEST(
+      _fp->c_from_v_e(v, e),
+      std::sqrt(bulk_modulus / _fp->rho_from_p_T(_fp->p_from_v_e(v, e), _fp->T_from_v_e(v, e))),
+      tol);
+  ABS_TEST(_fp->k_from_v_e(v, e), thermal_cond, tol);
+  ABS_TEST(_fp->s_from_v_e(v, e), entropy, tol);
+  ABS_TEST(_fp->T_from_v_e(v, e), _fp->T_from_v_e(v, e), tol);
+  ABS_TEST(_fp->p_from_v_e(v, e), _fp->p_from_v_e(v, e), tol);
```

Comment:
These 2 do not look right. Comparing `T_from_v_e ` to `T_from_v_e ` will always be true.

If you are testing p,T and e,v at the same time, you can do things like this:
```
p = value
T = value
rho = rho_from_p_T(p, T)
v = 1 / rho
e = e_from_p_T(p, T) 

ABS_TEST(p_from_v_e(v, e), p, tol)
```
The same for `T`.

But, these 2 tests should make sure that you retrieve the value of `p` and `T` use to compute `v` and `e`.

Reply:
@andrsd 👍 my bad, I modified `T` accordingly. However, for `P`, since it was hard to provide the exact number with decimals that can be under the tolerance of 1E-10, I just provided the equation for computing `P` from v and e to perform the unit test for this property. Please, let me know if this would work. Thank you!

Reply:
That works too. The important bit is that the values you are comparing have to be computed in 2 independent ways.

Reply:
Or that you know what the value should be (think of the case where you would compare to experimentally measured values).

Reply:
@andrsd that makes sense. :) thank you for reviewing the code!